scripts/release/create-release.hs

```diff
@@ -30,12 +30,13 @@ import           Data.Semigroup (Max (..))
 import qualified Data.Text as Text
 import qualified Data.Text.IO as Text
 import           Data.Version
+import           Main.Utf8 (withStdTerminalHandles)
 import           Prettyprinter
 import           System.FilePath
 import           Turtle hiding (d, fp, l, o)
 
 main :: IO ()
-main = sh do
+main = withStdTerminalHandles $ sh do
```

Comment:
You also have to adapt the `build-depends` here at the top of the file.

Reply:
Done in 2fd5bfa4d25f7e73391396be69aef48569f61bf9.